subprojects/gst-plugins-bad/sys/msdk/gstmsdkvpp.c

```diff
@@ -783,40 +783,89 @@ gst_msdkvpp_propose_allocation (GstBaseTransform * trans,
 }
 
 static GstMsdkSurface *
-get_surface_from_pool (GstMsdkVPP * thiz, GstBufferPool * pool,
-    GstBufferPoolAcquireParams * params)
+gst_msdkvpp_get_surface_from_sys (GstMsdkVPP * thiz, GstBufferPool * pool,
```

Comment:
I prefer to change this func's name to as similar one as before because this function does not actually get surface from system memory, it first acquires a buffer from its own bufferpool then copies the data from inbuf (could be system memory) to this buffer.